readthedocs/settings/celery_nodb.py

```diff
@@ -0,0 +1,25 @@
+"""Local setting for celery without DB access"""
+
+import os
+
+from .dev import CommunityDevSettings
+
+
+class CeleryDevSettings(CommunityDevSettings):
+    @property
+    def DATABASES(self):  # noqa
+        return {
+            'default': {}
+        }
+
+    DONT_HIT_DB = True
+
+
+CeleryDevSettings.load_settings(__name__)
+
+if not os.environ.get('DJANGO_SETTINGS_SKIP_LOCAL', False):
+    try:
+        # pylint: disable=unused-wildcard-import
+        from .local_settings import *  # noqa
+    except ImportError:
+        pass
```

Comment:
Local settings with a database defined will just override the database declaration above. We should either remove this or reset the database after local settings import.

Reply:
I moved this block before loading the settings from the class. This way, the class is overriding the other settings.

Reply:
This introduced another issue with setting load order. I've resolved with my work.